### PR TITLE
solves #300

### PR DESCRIPTION
**Pull request by lucasborin**
and improve its performance

### CODE REVIEW

File: src/checks/y_check_db_access_in_ut.clas.abap [hunk 1]
```diff
@@ -102,18 +104,21 @@ CLASS y_check_db_access_in_ut IMPLEMENTATION.
...
+    has_framework = abap_false.
+
     LOOP AT ref_scan_manager->statements ASSIGNING FIELD-SYMBOL(<statement>)
-    FROM class_definition-stmnt_from TO class_definition-stmnt_to.
-      IF is_in_scope( <statement> ) = abap_false.
+         FROM class_definition-stmnt_from TO class_definition-stmnt_to.
+
+      IF is_in_scope( <statement> ) = abap_false
+         OR test_risk_level IS NOT INITIAL
+         OR has_framework EQ abap_true.
```
Comment (lucasborin) [inline src/checks/y_check_db_access_in_ut.clas.abap:120]: not relevant anymore

File: src/checks/y_check_db_access_in_ut.clas.abap [hunk 2]
```diff
@@ -102,18 +104,21 @@ CLASS y_check_db_access_in_ut IMPLEMENTATION.
...
       CATCH cx_sy_itab_line_not_found.
         RETURN.
     ENDTRY.
 
+    has_framework = abap_false.
+
     LOOP AT ref_scan_manager->statements ASSIGNING FIELD-SYMBOL(<statement>)
-    FROM class_definition-stmnt_from TO class_definition-stmnt_to.
-      IF is_in_scope( <statement> ) = abap_false.
+         FROM class_definition-stmnt_from TO class_definition-stmnt_to.
```
Comment (lucasborin) [inline src/checks/y_check_db_access_in_ut.clas.abap:116]: ident

File: src/checks/y_check_db_access_in_ut.clas.abap [hunk 3]
```diff
@@ -32,7 +32,8 @@ CLASS y_check_db_access_in_ut DEFINITION PUBLIC INHERITING FROM y_check_base CRE
 ENDCLASS.
 
 
-CLASS y_check_db_access_in_ut IMPLEMENTATION.
+
+CLASS Y_CHECK_DB_ACCESS_IN_UT IMPLEMENTATION.
```
Comment (lucasborin) [inline src/checks/y_check_db_access_in_ut.clas.abap:36]: to lower

Comment (larshp) [reply src/checks/y_check_db_access_in_ut.clas.abap:36]: by default abaplint ignores this, but you can enable it, https://rules.abaplint.org/keyword_case/ try setting `ignoreLowerClassImplmentationStatement` to true

**Pull request merged**